xwiki-platform-tools/xwiki-platform-tool-jetty/xwiki-platform-tool-jetty-resources/src/main/resources/start_xwiki.sh

```diff
@@ -204,20 +204,23 @@ java_version() {
 
 # Check version of Java (when in non-interactive mode)
 JAVA_VERSION="$(java_version)"
-if [ ! "$XWIKI_NONINTERACTIVE" = true ] ; then
-  if [[ "$JAVA_VERSION" -eq "no_java" ]]; then
-    echo "No Java found. You need Java installed for XWiki to work."
-    exit 0
-  fi
-  if [ "$JAVA_VERSION" -lt 11 ]; then
-    echo This version of XWiki requires Java 11 or greater.
-    exit 0
-  fi
-  if [ "$JAVA_VERSION" -gt 17 ]; then
+
```

Comment:
I would not have added this extra new line so that the code is nicely under the comment  (i.e. `# Check version of Java (when in non-interactive mode)`).

Also, the comment is no longer valid since you removed the check for `$XWIKI_NONINTERACTIVE`.